packages/http-server/src/getHttpConfigFromRequest.ts

```diff
@@ -1,14 +1,18 @@
 import { PrismConfig, PrismConfigFactory, resolveConfig } from '@stoplight/prism-core';
 import { IHttpConfig, IHttpRequest } from '@stoplight/prism-http';
 
-export const getHttpConfigFromRequest: PrismConfigFactory<IHttpConfig, IHttpRequest> = async (
+export const getHttpConfigFromRequest: PrismConfigFactory<IHttpConfig, IHttpRequest> = (
   req: IHttpRequest,
   defaultConfig?: PrismConfig<IHttpConfig, IHttpRequest>
 ) => {
   // For some reason this fixed the code coverage.
   let config: IHttpConfig = { mock: true };
+
   if (defaultConfig) {
-    config = await resolveConfig<IHttpConfig, IHttpRequest>(req, defaultConfig);
+    config = Object.assign(
+      { mock: true },
```

Comment:
Maybe reuse the config from line 9?

```suggestion
      config,
```

Reply:
Fair enough, although it's a little bit confusing. Done!